tools/orca/src/state.rs

```diff
@@ -1,11 +1,12 @@
 use crate::error::Error;
 use crate::model::ActorModel;
+use crate::models::markov::DISTR_MATRIX_SIZE;
 use crate::profile::Profile;
 use serde::{Deserialize, Serialize};
 use std::collections::BTreeSet;
 use std::path::Path;
 
-/// A serialisable state representing the content of a kanidm database and potential
+/// A serializable state representing the content of a kanidm database and potential
```

Comment:
English AU spelling, it was correct before.

Reply:
Wops my bad